uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserEdge.java

```diff
@@ -51,22 +53,10 @@ public BrowserEdge() {
 
     @Override
     public void handleFirstRun() {
-        if (shouldHandleFirstRun) {
-            Logger.i(TAG, "Handle First Run of Browser..");
-            // cancel sync in Edge
-            UiAutomatorUtils.handleButtonClick("com.microsoft.emmx:id/not_now");
-            sleep(); // need to use sleep due to Edge animations
-            // cancel sharing data
-            UiAutomatorUtils.handleButtonClick("com.microsoft.emmx:id/not_now");
-            sleep(); // need to use sleep due to Edge animations
-            // cancel personalization
-            UiAutomatorUtils.handleButtonClick("com.microsoft.emmx:id/fre_share_not_now");
-            sleep();// need to use sleep due to Edge animations
-            // avoid setting default
-            UiAutomatorUtils.handleButtonClick("com.microsoft.emmx:id/no");
-            sleep();// need to use sleep due to Edge animations
-            shouldHandleFirstRun = false;
-        }
+        Logger.i(TAG, "Handle First Run of Browser..");
+        // cancel sync in Edge
+        UiAutomatorUtils.handleButtonClickForObjectWithText("Not now");
+        sleep(); // need to use sleep due to Edge animations
```

Comment:
Is this an accurate change? I feel like there may be some scenarios where we would get more than just 1 page with "Not now" button. Maybe just double check that other test cases that use Edge are still passing with this change, we don't want to break existing automation.

Reply:
If anything, why are we removing the conditional check for shouldHandleFirstRun, and subsequent setting "Handle First Run" to false once we finish handling first run.

Reply:
Let's make sure the existing cases using edge are still passing. If so, we can merge this change in to also allow this case to pass.